python/BioSimSpace/Sandpit/Exscientia/Convert/_convert.py

```diff
@@ -225,8 +225,7 @@ def to(obj, format="biosimspace", property_map={}):
                 space = _SireVol.Cartesian()
 
             # Set a shared space property.
-            obj._sire_object.addSharedProperty(prop)
```

Comment:
@lohedges Is this the intended change?

Reply:
Yes. I'm not sure if it is still needed, but the conversion to OpenMM within Sire would not pick up the space property unless as added as a shared system property. Is this causing problems?

Reply:
This was introduced as part of [this](https://github.com/OpenBioSim/biosimspace/pull/160) PR.

Reply:
Oh sorry, I see this is a deletion. This is a problem with the way that I need you so the synchronization, i.e. taking your fork as the truth. Due to the way that you pull in updates from `main` it means that some of my fixes get replaced since they are less recent. This file should be the same as the core code.

Reply:
This is still present in the Sandpit on `devel` and `main` of our repo, so why is it being deleted here?

Reply:
Oh, you've just commented on a single line, not the whole block related to this change so it makes no sense. The change as a whole does, which was to remove adding then setting the shared property, since adding it automatically sets it. Sorry for the confusion, but it wasn't at all clear from this single line